### PR TITLE
Authorization + Collection Interface AST

### PR DESCRIPTION
**Pull request by mateuszmlc**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -300,41 +346,44 @@ mod tests {
...
-            _ => panic!("Expected if"),
-        };
-
-        assert!(
-            matches!(if_.condition, ast::Expression::Equal(n, m) if *n == ast::Expression::Primitive(ast::Primitive::Number(1.0)) && *m == ast::Expression::Primitive(ast::Primitive::Number(1.0)))
-        );
-        assert_eq!(if_.then_statements.len(), 1);
-        assert_eq!(if_.else_statements.len(), 1);
-    }
+    // #[test]
```
Comment (calummoore) [inline src/lib.rs:349]: Any reason this is commented out? Are we no longer supporting fns at the root, or was that never supported?

Comment (mateuszmlc) [reply src/lib.rs:349]: I added them during the initial rewrite of the parser from Javascript, but I don't think we support them in the indexer, so I thought it's better to disallow them for now.

Comment (calummoore) [reply src/lib.rs:349]: We can specifically add them back in at some point then, np leaving them out for now!

**Review by calummoore — APPROVED** [review]

**Comment by calummoore** [comment]
@mateuszmlc do you want to publish a new version and update `polybase` / `polybase-ts`

**Pull request merged**